packages/core/src/database/models/learnAndEarn/learnAndEarnPayment.ts

```diff
@@ -42,7 +44,7 @@ export function initializeLearnAndEarnPayment(
             levelId: {
                 type: DataTypes.INTEGER,
                 references: {
-                    model: 'learn_and_earn_lesson',
+                    model: learnAndEarnLesson,
```

Comment:
actually, this column should reference `learnAndEarnLevel`, not lesson (it was wrong before too)